plugin/trino-databend/src/main/java/io/trino/plugin/databend/DatabendConnectionFactory.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.databend;
+
+import io.trino.plugin.jdbc.ConnectionFactory;
+import io.trino.plugin.jdbc.ForwardingConnection;
+import io.trino.spi.connector.ConnectorSession;
+import jakarta.annotation.PreDestroy;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import static java.util.Objects.requireNonNull;
+
+public class DatabendConnectionFactory
```

Comment:
Please leave a comment why this class is needed.

Reply:
Yes we don't need this class. I will remove it.